cpp/src/parquet/parquet.thrift

```diff
@@ -1070,6 +1275,8 @@ struct ColumnIndex {
     **/
    7: optional list<i64> definition_level_histograms;
 
+   /** A list containing statistics of GEOMETRY logical type for each page */
+   8: optional list<GeometryStatistics> geometry_stats;
```

Comment:
Just want to remind that `ColumnIndex` has a per-page geometry stats. It would be good to complete this as well.

Reply:
I've added geometry stats to `ColumnIndex`. I'm not sure how to set `min_values` and `max_values` for column indexes of geometry columns. Since the sort order is undefined, we assume that the geometry column does not have min/max bounds, so I just left them as empty lists.